xbmc/pvr/PVRManager.cpp

```diff
@@ -365,6 +365,7 @@ void CPVRManager::SetState(ManagerState state)
     CSingleLock lock(m_managerStateMutex);
     m_managerState = state;
     SetChanged();
+    TriggerEpgsCreate();
```

Comment:
Calling this on every (!) pvr manager state change looks horribly wrong to me (although in fact the existing code does eaxtly this). So, your change does not change behavior, but I doubt that the existing behavior is okay. We need to find a good condition for when to call TriggerEpgsCreate...